evm_loader/shared-lib/src/lib.rs

```diff
@@ -0,0 +1,233 @@
+use abi_stable::{
+    export_root_module,
+    prefix_type::WithMetadata,
+    sabi_extern_fn,
+    std_types::{RResult, RStr, RString},
+    DynTrait,
+};
+use async_ffi::{BorrowingFfiFuture, FutureExt};
+use neon_interface::{
+    types::{BoxedConfig, BoxedContext, NeonLibError, RNeonResult},
+    NeonLib, NeonLib_Ref,
+};
+use neon_lib::{
+    commands::{
+        cancel_trx, collect_treasury, create_ether_account, deposit, emulate,
+        get_ether_account_data, get_neon_elf, get_storage_at, init_environment,
+    },
+    config::create_from_api_comnfig,
+    context::{build_hash_rpc_client, build_rpc_client},
+    signer::NeonSigner,
+    Config, Context, NeonError,
+};
+
+const _MODULE_WM_: &WithMetadata<NeonLib> = &WithMetadata::new(NeonLib {
+    hash,
+    get_version,
+    init_config,
```

Comment:
We should avoid of using config, context etc. between Neon Core API and Neon Library. Pass just needed signer's pubkey, evm-loader address, etc. to the exported functions.